Azure/sonic-buildimage
# [201911][Mellanox] Update SDK to 4.4.2318, FW to *.2008.2312

Signed-off-by: Volodymyr Samotiy <volodymyrs@nvidia.com>



**- Why I did it**

To have the following fixes:
* All | Port status remains down after warm boot and flapping the port on peer side
* All | LAG HASH  | IPv6 SRC_IP is not accounted in LAG hashing [
* All | ASIC driver | Kernel crash observed when driver reload is initiated before it fully loaded
* Spectrum-3 | Buffer | In lossless configuration, headroom is been evicted only when the shared buffers is free

**- How I did it**
Updated SDK submodule and version in appropriate Makefiles

**- How to verify it**
Build an image, deploy to the switch and verify all is up and running

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**

[201911][Mellanox] Update SDK to 4.4.2318, FW to *.2008.2312

**- A picture of a cute animal (not mandatory but encouraged)**
